v2/pkg/engine/resolve/resolvable.go

```diff
@@ -388,18 +388,19 @@ func (r *Resolvable) printValueCompletionExtension() error {
 }
 
 func (r *Resolvable) hasExtensions() bool {
-	if r.ctx.authorizer != nil && r.ctx.authorizer.HasResponseExtensionData(r.ctx) {
+	if !r.ctx.AuthorizerOptions.SkipPrintExtension && r.ctx.authorizer != nil && r.ctx.authorizer.HasResponseExtensionData(r.ctx) {
 		return true
 	}
-	if r.ctx.RateLimitOptions.Enable && r.ctx.RateLimitOptions.IncludeStatsInResponseExtension && r.ctx.rateLimiter != nil {
+	if !r.ctx.RateLimitOptions.SkipPrintExtension && r.ctx.RateLimitOptions.Enable && r.ctx.RateLimitOptions.IncludeStatsInResponseExtension && r.ctx.rateLimiter != nil {
```

Comment:
@StarpTech fyi Ive gone with the approach where there are two flags for this and RateLimiting, by doing this there will be a central place in the config just for extensions.

I chose not to merge the flags internally to make it obvious that there are two flags for this.